pages/common/jrnl.md

```diff
@@ -0,0 +1,23 @@
+# jrnl
+
+> A simple journal application for your command line.
+
+- Insert a new entry with your editor:
+
+`jrnl`
+
+- Quickly insert a new entry:
+
+`jrnl {{timestamp}}: {{title}}. {{content}}`
```

Comment:
"timestamp" is not immediately apparent as to the format of the timestamp to be used. I think it's better we use a concrete example for this.